tsconfig.test.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "./tsconfig.json",
+  "include": ["e2e/**/*"],
+  "ts-node": {
+    "compilerOptions": { "module": "CommonJS" }
+  }
```

Comment:
Is this needed? The root `tsconfig.json` from which this config extends already has these `ts-node` options, I believe.